static/js/entry/zendesk_widget.js

```diff
@@ -116,6 +118,46 @@ const zendeskCallbacks = {
         }, 100);
       };
     }
+  },
+  ticketSubmissionFormLoaded: () => {
+    const iframe = document.querySelector("iframe.zEWidget-ticketSubmissionForm");
+
+    const fieldSelector = name => (
+      `input[name="${name}"], select[name="${name}"]`
+    );
+    const iframeQuerySelector = query => (
+      iframe.contentDocument.querySelector(query)
+    );
+    const fieldElement = R.compose(
+      iframeQuerySelector, fieldSelector
+    );
```

Comment:
given the workaround here it might be a little more elegant to just do:

```js
const fieldElement = name => (
  iframe.contentDocument.querySelector(fieldSelector(name))
);
```

up to you though!